apps/codeforafrica/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,35 @@
+import { Section, RichTypography } from "@commons-ui/core";
+import { Box } from "@mui/material";
+import React from "react";
+
+import TwoToneBackground from "../TwoToneBackground";
+
+const PageHeader = React.forwardRef(function PageHeader(props, ref) {
+  const { title, description } = props;
+  if (!title && !description) {
+    return null;
+  }
+  return (
+    <Box ref={ref} sx={{ backgroundColor: "background.main" }}>
+      <Section>
+        <TwoToneBackground>
+          <Box sx={{ py: 9.6, zIndex: 1, textAlign: "center" }}>
+            <RichTypography
+              sx={{
+                textTransform: "uppercase",
```

Comment:
Define a `h5ExtraBold` variant in theme.

![Screenshot from 2022-06-08 09-38-00](https://user-images.githubusercontent.com/1779590/172548775-576bf985-adf1-4f4f-9669-c9e8cc6e0cd9.png)